### PR TITLE
IDENTITY-6965: Intermitent issue in SCIM group creation

### PR DESCRIPTION
**Pull request by IsurangaPerera**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: modules/charon-core/src/main/java/org/wso2/charon3/core/objects/Group.java [hunk 1]
```diff
@@ -83,8 +84,11 @@ public void setDisplayName(String displayName) throws CharonException, BadReques
             for (Attribute subValue : subValuesList) {
                 ComplexAttribute complexAttribute = (ComplexAttribute) subValue;
                 Map<String, Attribute> subAttributesList = complexAttribute.getSubAttributesList();
-                memberList.add(((SimpleAttribute) (subAttributesList.get(
-                                SCIMConstants.CommonSchemaConstants.VALUE))).getValue());
+                if(subAttributesList != null && subAttributesList.containsKey(
```
Comment (IsuraD) [inline modules/charon-core/src/main/java/org/wso2/charon3/core/objects/Group.java]: Formatting issue

File: modules/charon-core/src/main/java/org/wso2/charon3/core/objects/Group.java [hunk 2]
```diff
@@ -107,8 +111,12 @@ public void setDisplayName(String displayName) throws CharonException, BadReques
                 for (Attribute subValue : subValuesList) {
                     ComplexAttribute complexAttribute = (ComplexAttribute) subValue;
                     Map<String, Attribute> subAttributesList = complexAttribute.getSubAttributesList();
-                    displayNames.add(((SimpleAttribute) (subAttributesList.get(
-                            SCIMConstants.CommonSchemaConstants.DISPLAY))).getValue());
+                    if(subAttributesList != null && subAttributesList.containsKey(
```
Comment (IsuraD) [inline modules/charon-core/src/main/java/org/wso2/charon3/core/objects/Group.java]: formatting issue

File: modules/charon-core/src/main/java/org/wso2/charon3/core/objects/Group.java (hunk 1 above)
Comment (IsurangaPerera) [reply modules/charon-core/src/main/java/org/wso2/charon3/core/objects/Group.java]: @IsuraD  can you bit explain about the formatting issue

Comment (IsuraD) [reply modules/charon-core/src/main/java/org/wso2/charon3/core/objects/Group.java]: There should be a space after the "if condition" as follows. 

if (subAttributesList != null && subAttributesList.containsKey(

**Pull request merged**